pkg/injector/sidecar/container_test.go

```diff
@@ -172,7 +172,6 @@ func TestGetSidecarContainer(t *testing.T) {
 			"--dapr-http-read-buffer-size", "-1",
 			"--dapr-graceful-shutdown-seconds", "-1",
 			"--disable-builtin-k8s-secret-store=false",
-			"--app-channel-address", "",
```

Comment:
Look around the code, we do not have a test for no-empty app-channel-address.

Reply:
Let me merge this and will add the new test scenario. It is blocking another PR that requires this fix to have E2E tests with old images.